### ISSUE TITLE
modofu dev build

### ISSUE THREAD

**Issue by vladislavs1321** [modofu-org/modofu#2]
will be nice to have possibility to have a parameter like --name=service, to build on or two services, not a whole stack, and automatically remove old ones.

**Comment by nex-dabe** [comment]
Thanks for your proposal. It for sure is a good idea to add a `--service` parameter to `dev build`.
But can you give some explanation, what do you mean with *automatically remove old ones*?

**Comment by vladislavs1321** [comment]
when i build new service image i want to run service container based on newest version of image, so i need to stop old one, remove old one, and create. Is it possible to automate stop-remove-create process?

**Comment by nex-dabe** [comment]
So something like a `modofu dev build --service=webserver` would be enough? If so, this should be easy to do...

**Comment by nex-dabe** [comment]
@vladislavs1321 modofu has now support for building individual services:
```
# modofu dev build --help
Usage: # modofu dev build [options] [SERVICE...]

Options:
    --force-rm       Always remove intermediate containers.
    --no-cache       Do not use cache when building the image.
    --no-pull        Do not refresh intermediate images.
```
> Note: The incorrect description of option `--no-pull` was fixed in 1466e7fcfc8d2963b583c40f46212806ed771517

Of course multiple services are supported as well - e.g.: `modofu dev build webserver redis`

**Comment by nex-dabe** [comment]
(Available since [2017-12-19T12:31:02.144Z](https://hub.docker.com/r/modofu/modofu/builds/bnfzsppjwrkkqszkvdzprj9/))
